spec/options/options_validation.spec.js

```diff
@@ -26,12 +26,30 @@ describe('validation options', function() {
       }});
 
       var validate = ajv.compile({ format: 'identifier' });
+
       validate('Abc1') .should.equal(true);
       validate('123') .should.equal(false);
       validate(123) .should.equal(true);
     });
   });
 
+  describe('keywords', function() {
+    it('should add keywords from options', function() {
+      var ajv = new Ajv({ keywords: {
+        identifier: {
```

Comment:
I suggest adding type: "string" in the definition and changing the implementation to `/^[a-z_$][a-z0-9_$]*$/i.test(data)` so the example in tests makes sense. Then the same tests can be used as in formats option test.